docs/nddata/nddata.rst

```diff
@@ -98,18 +98,7 @@ or a `numpy.ma.MaskedArray`::
     >>> ndd4.data
     array([ 5, 10, 15])
     >>> ndd4.mask
-    array([False,  True, False], dtype=bool)
-
-or even a masked Quantity::
```

Comment:
I don't think we should encourage this in any way! Also, it looks very different in numpy 1.14 (probably a buglet)

Reply:
Thanks for this feedback!